extender/scheduler.go

```diff
@@ -146,6 +150,7 @@ func configureSecureServer(port string, caFile string) *http.Server {
 		},
 	}
 
+	//nolint:gomnd
```

Comment:
Maybe we can exclude this package from linting in some other way - I can see these tags becoming too tempting / difficult to maintain and make consistent over time. My preference would be to just not run gomnd :)

Reply:
I will limit gomnd to one block per file, at the top